corehq/apps/es/aggregations.py

```diff
@@ -111,6 +111,28 @@ def counts_by_bucket(self):
         return {b['key']: b['doc_count'] for b in self.normalized_buckets}
 
 
+class TopHitsResult(AggregationResult):
```

Comment:
this is pretty similar to ESQuerySet, but seemed messy to try and be completely DRY. not that much copied code